components/components.css

```diff
@@ -0,0 +1,591 @@
+
+/* Images */
+.img-container {
+  display: flex;
+  justify-content: center;
+  margin: 2rem 0;
+}
+
+.img-responsive {
+  width: 40%;
+  margin: 0 2rem;
+}
+
+.img-circular {
+  margin: 0 1rem;
+  border-radius: 50%;
+}
+
+.img-small {
+  width: 75px;
+  height: 75px;
+}
+
+.img-medium {
+  width: 100px;
+  height: 100px;
+}
+
+.img-large {
+  width: 150px;
+  height: 150px;
+}
+
+/* Input */
+
+.input-container {
+  position: relative;
+  margin: 1rem 0;
+}
+
+.input-container .input-normal {
+  padding: 0.75rem 1rem;
+  border: 1px solid gray;
+  outline: none;
+  border-radius: 0.5rem;
+  width: 20%;
+}
+
+.input-normal:focus,
+.input-normal:valid {
+  border: 2px solid var(--primary-color);
+}
+
+.input-container label {
+  position: absolute;
+  left: 0.75rem;
+  top: 0.75rem;
+  font-size: 14px;
+  background-color: white;
+  color: var(--gray);
+  transition: all 0.2s ease-in-out;
+}
+
+.input-normal:focus + label,
+.input-normal:valid + label {
+  top: -8px;
+  padding: 0 0.2rem;
+  color: var(--primary-color);
+}
+
+.input-container .input-error {
+  padding: 0.75rem 1rem;
+  border: 1px solid red;
+  outline: none;
+  border-radius: 0.5rem;
+  width: 20%;
+}
+
+.input-error:focus,
+.input-error:valid {
+  border: 2px solid red;
+}
+
+.input-error:focus + label,
+.input-error:valid + label {
+  top: -8px;
+  padding: 0 0.3rem;
+  color: red;
+}
+
+.input-container .label-error {
+  color: #f87171;
+}
+
+/* Navigation */
+
+.navbar {
+  background-color: var(--primary-color);
+  padding: 0.5rem 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: white;
+  border-radius: 1rem;
+}
+
+.navbar .nav-brand {
+  font-size: 1.2rem;
+  font-weight: 550;
+  cursor: pointer;
+}
+
+.list-non-bullet {
+  list-style: none;
+  padding-inline-start: 0;
+}
+
+.list-item-inline {
+  display: inline;
+  margin: 0 0.75rem;
+}
+
+.navbar .link {
+  color: white;
+  text-decoration: none;
+}
+
+.navbar .link:hover {
+  border: 1px solid var(--primary-color);
+  background-color: #3b82f6;
+  padding: 0.3rem 0.75rem;
+  border-radius: 0.5rem;
+  font-weight: bold;
+}
+
+.navbar .link-active {
+  border: 1px solid var(--primary-color);
+  background-color: #3b82f6;
+  padding: 0.5rem 1rem;
+  border-radius: 0.5rem;
+  font-weight: bold;
+}
+
+.modal {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
+  max-width: 70vh;
+  height: auto;
+  bottom: 5%;
+  margin: 2.5rem 0 1rem;
+}
+
+.modal .modal-title {
+  display: flex;
+  background-color: black;
+  color: white;
+  padding: 1rem 0.5rem;
+  font-size: 1.25rem;
+}
+
+.modal-title > .btn-close {
+  position: absolute;
+  top: 1.3rem;
+  right: 1rem;
+  border: none;
+  background-color: none;
+  outline: none;
+}
+
+.modal-body {
+  padding: 1rem;
+}
+
+.modal-body button {
+  background-color: black;
+  color: white;
+  border: none;
+  outline: none;
+  padding: 0.5rem 1rem;
+  margin: 1rem 0rem;
+  font-size: 1rem;
+}

```

Comment:
add cursor as pointer and avoid to set outline as none.

Reply:
Added ``` cursor: pointer ``` property and removed the ``` outline: none ``` property. Thanks.